_posts/2020-03-13-status-update.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "13th March 2020: IAC Team Status Update"
+author: carabasdaniel
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - community
+  - litmus
+  - facter
+  - testing
+  - pdk
+---
+
+The IAC team has been working on a lot of interesting things over the past week.
+Here's a brief overview of what we've been up to:
+
+## Updates to Puppet Modules
+
+Over the last week we have released:
+
+- [puppetlabs/apt](https://forge.puppet.com/puppetlabs/apt) (v7.4.1)
+
+## Community
+
+A big thank you goes out to Glen Sarti for the work he has done in removing the PDK dependecy from Litmus. Also we would like to thank [Antoine](https://github.com/anarcat) and [Dirk Heinrichs]() for their help in resolving the file permission issues on the [puppetlabs-apt](https://github.com/puppetlabs/puppetlabs-apt) module.
```

Comment:
```suggestion
A big thank you goes out to [Glenn Sarti](https://github.com/glennsarti) for the work he has done in removing the PDK dependency from Litmus.
Also we would like to thank [Antoine](https://github.com/anarcat) and [Dirk Heinrichs](https://tickets.puppetlabs.com/secure/ViewProfile.jspa?name=dhs%40recommind.com&selectedTab=com.atlassian.streams.streams-jira-plugin:user-profile-stream-panel) for their help in resolving the file permission issues on the [puppetlabs-apt](https://github.com/puppetlabs/puppetlabs-apt) module.
```
* link to Glenn's profile
* link to Dirk's profile
* spell check
* semantic linebreaks